Learning arrays

emiro66 opened the pull request.

emiro66 committed:
добавила коммент по мапам

emiro66 committed:
add "//" in arrays_slices\main.go
add "//" in slicesPractice\main.go
completed all the tasks in slicesPractice\main.go

bobisdacool1 commented on slicesPractice/main.go:
```diff
@@ -0,0 +1,176 @@
+}
+
+func findMinMax(numbers []int) (int, int) { //находит максимум и минимум
+	max := numbers[0]
```
Если входящий слайс пустой, мы получим панику

bobisdacool1 commented on slicesPractice/main.go:
```diff
@@ -0,0 +1,176 @@
+			result = append(result, s) // то добавляется
+		}
+	}
+	return
```
Результат не возвращается

bobisdacool1 commented on slicesPractice/main.go:
```diff
@@ -0,0 +1,176 @@
+}
+
+// строки длиной больше 3 символов, начинаются с заглавной буквы
+func strings(input []string) (result []string) {
```
Давай пока уберем именованое возаращаемое значение. оно редко нужно

bobisdacool1 commented on slicesPractice/main.go:
```diff
@@ -0,0 +1,176 @@
+
+}
+
+func ballMap() {
```
карта яиц

bobisdacool1 commented on slicesPractice/main.go:
```diff
@@ -0,0 +1,176 @@
+func deleteStudent(studentMap map[string]int, name string, score int) (map[string]int, error) {
+	_, exists := studentMap[name]
+	if !exists {
+
```
ошибка форматирования

bobisdacool1 reviewed (changes requested):
надо доработать

emiro66 committed:
changed the name of the map to "scoresMap"
removed it from the variables "score int"
added a check for an incoming empty slice in "findMinMax"
in function "strings" returns the result
